apps/mobile/src/i18n/locales/pt-BR.json

```diff
@@ -40,5 +46,8 @@
       "GENDER_REQUIRED": "Informe o sexo do seu Pet",
       "SIZE_REQUIRED": "Informe o porte do seu Pet"
     }
+  },
+  "PERMISSIONS": {
+    "CAMERA": "Animavita precisa do acesso a câmera"
```

Comment:
Please take a look at my comment above about getting rid of alerts. Then, here, let's come up with a friendly message like "Hey, in order to have better chances of finding a new owner for your pet, we need access to your gallery so you can upload some nice pictures".

Reply:
Actually, you can just remove this.